hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestBucketRequest.java

```diff
@@ -69,6 +69,9 @@ public void setup() throws Exception {
     OzoneConfiguration ozoneConfiguration = new OzoneConfiguration();
     ozoneConfiguration.set(OMConfigKeys.OZONE_OM_DB_DIRS,
         folder.newFolder().getAbsolutePath());
+    ozoneConfiguration.set(OMConfigKeys.OZONE_DEFAULT_BUCKET_LAYOUT,
+        OMConfigKeys.OZONE_BUCKET_LAYOUT_FILE_SYSTEM_OPTIMIZED);
```

Comment:
It may be useful to declare the `ozoneConfiguration` variable at the class level as a protected variable.  Then allow the separate tests that extend the class to change the default layout for the specific tests.  In this case for the `TestBucketRequest` the layout can be set to `OZONE_DEFAULT_BUCKET_LAYOUT_DEFAULT `for consistency with the default `OMConfigKeys`.  Then in the tests it can be set when needed.  For FSO specific tests (`TestOMBucketCreateRequestWithFSO.java`), we can set the protected OzoneConfiguration to FSO layout like:
`ozoneConfiguration.set(OMConfigKeys.OZONE_BUCKET_LAYOUT_FILE_SYSTEM_OPTIMIZED)`.

Reply:
@neils-dev We don't need to make `ozoneConfiguration` a class variable because we are accessing it in the subclasses through `ozoneManager.getConfiguration()`. I made the changes you suggested.

Reply:
Thanks @xBis7 for the patch and for the changes.  LGTM.

Reply:
Thanks @xBis7 for the PR updates and @sadanand48 for your review and comments.  Re-running workflow jobs for a green build prior to closing.